editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -1070,6 +1070,7 @@ VersionControlEditorPlugin::VersionControlEditorPlugin() {
 	set_up_ssh_public_key_path->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	set_up_ssh_public_key_path->set_accessibility_name(TTRC("SSH Public Key Path"));
 	set_up_ssh_public_key_path->set_text(EDITOR_GET("version_control/ssh_public_key_path"));
+	set_up_ssh_public_key_path->set_placeholder(TTR("~/.ssh/id_rsa.pub"));
```

Comment:
A path is never translatable I think. They same applies to the private key path placeholder.